.github/workflows/documentation.yml

```
@@ -0,0 +1,25 @@
+name: Docs
+
+on:
+  push:
+    branches: [ stable ]
+    paths:
+      - 'docs/**'
+      - 'mkdocs.yml'
```

Also run workflow when the workflow is updated
```suggestion
    paths:
      - 'docs/**'
      - 'mkdocs.yml'
      - '.github/workflows/documentation.yml'
```